src/mvContainers.cpp

```diff
@@ -57,6 +57,8 @@ DearPyGui::fill_configuration_dict(const mvChildWindowConfig& inConfig, PyObject
     PyDict_SetItemString(outDict, "border", mvPyObject(ToPyBool(inConfig.border)));
     PyDict_SetItemString(outDict, "autosize_x", mvPyObject(ToPyBool(inConfig.autosize_x)));
     PyDict_SetItemString(outDict, "autosize_y", mvPyObject(ToPyBool(inConfig.autosize_y)));
+    PyDict_SetItemString(outDict, "x_scroll_pos", mvPyObject(ToPyFloat(inConfig.scrollX)));
```

Comment:
Just curious, doesn't `dpg.get_x_scroll()` do just the same? Each new value added to `fill_configuration_dict` will slow down (just a bit) a lot of functions that internally use `dpg.get_item_configuration()`.